### PR TITLE
fixed fresh item containing lots of codes, caves 

### LINKED ISSUE #644: Fresh `item` contains lots of codes, caves, handles.
When I add an item to a batch, the new item has lots of codes, caves, handles pre-assigned.  They should be empty.

screencast

### PR DESCRIPTION
**Pull request by TahaKhanAbdalli**
fixes #644

### CODE REVIEW

File: src/components/ProtectionBlockInputs.tsx [hunk 1]
```diff
@@ -46,27 +45,6 @@ export default function ProtectionBlockInputs<
     })
   }
```
Comment (IanMayo) [inline src/components/ProtectionBlockInputs.tsx:47]: @TahaKhanAbdalli - can you explain why we've deleted the following block of code please?

**Review by IanMayo — APPROVED** [review]
Tested and working

**Review by IanMayo — CHANGES_REQUESTED** [review]
Aaah, there is still a pending debug line in the second file.

Could you also add a comment explaining why we're checking for mock env variable in the second file please @TahaKhanAbdalli ?

**Comment by TahaKhanAbdalli** [comment]
> Aaah, there is still a pending debug line in the second file.
> 
> Could you also add a comment explaining why we're checking for mock env variable in the second file please @TahaKhanAbdalli ?

We are checking for mock data because `mock` accepts `undefined` and `SQLite` accepts `null`

File: src/components/ProtectionBlockInputs.tsx (hunk 1 above)
Comment (TahaKhanAbdalli) [reply src/components/ProtectionBlockInputs.tsx:47]: This is `pMarking`, that we discussed with salony and it needed to be removed becuase our db has protectiveMarking not pMarkin.

**Comment by IanMayo** [comment]
> We are checking for mock data because `mock` accepts `undefined` and `SQLite` accepts `null`

Aah, ok. It will be good when [this](issue us resolved), and we just have `null`.

**Review by IanMayo — APPROVED** [review]

**Pull request merged**